tests/js/lib/request.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+var fluid = require("infusion");
+
+var kettle = require("kettle");
+kettle.loadTestingSupport();
+
+fluid.defaults("gpii.express.tests.request", {
```

Comment:
Could you follow the project standard ordering and place this instead in "gpii.test.express.request"? Cheers